src/main/java/org/micks/DiscGolfApplication/events/DiscGolfEventsController.java

```diff
@@ -71,4 +71,18 @@ public void registerUserForEvent(@RequestHeader(value = "Authorization") String
         UserDTO userDTO = userService.getLoggedInUser(authorizationHeader);
         eventRegistrationService.registerUserForEvent(userDTO.getUserId(), eventId);
     }
+
+    @GetMapping("/my-events")
+    public ResponseEntity<List<String>> getMyEventIds(
+            @RequestHeader(value = "Authorization") String authorizationHeader) {
+        log.info("Fetching registered event IDs for user");
+        try {
+            UserDTO userDTO = userService.getLoggedInUser(authorizationHeader);
+            List<String> myEventIds = eventRegistrationService.getMyEvents(userDTO.getUserId());
+            return ResponseEntity.ok(myEventIds);
+        } catch (Exception e) {
```

Comment:
We don't need this catch block all. Any error will be logged and mapped to 500 internal server error anyway.

Reply:
Fixed